library/general/disallowlatest/template.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sdisallowlatest
+  annotations:
+    description: Requires container images to contain an image tag different that "latest".
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sDisallowLatest
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+        package k8sdisallowlatest
+
+        violation[{"msg": msg}] {
+          container := input.review.object.spec.containers[_]
+          latest := [re_match(":latest", container.image)]
```

Comment:
Performing a suffix match might be safer here. Either that or we should anchor the regex to the end of the string using "$", though we should strip whitespace if we do that.